utils/protobuf-schema-utilities/src/test/proto/TestOrderingSyntax3References.proto

```diff
@@ -68,3 +68,27 @@ enum AddressType {
 message AnotherType {
     message Address {}
 }
+
+service OriginalService {
+  rpc Foo(Customer) returns(Address);
+}
+
+service DeprecatedService {
+  rpc Foo(Customer) returns(Address);
+  option deprecated = true;
+}
+
+service DeprecatedMethod {
```

Comment:
Can you add some messages in between the services?

Reply:
Updated